python/paddle/fluid/optimizer.py

```diff
@@ -1978,6 +2001,10 @@ def _append_average_accumulate_op(self, param):
     @signature_safe_contextmanager
     def apply(self, executor, need_restore=True):
         """Apply average values to parameters of current model.
+
+        Args:
+            executor(fluid.Executor): current executor.
+            need_restore(bool): If you finally need to do restore, set it to True.
```

Comment:
Please say the default value of `need_restore ` in this doc.

Reply:
thanks, fixed